src/Raven.Server/Documents/Revisions/RevisionsStorage.cs

```diff
@@ -670,11 +673,46 @@ public class DeleteOldRevisionsResult
 
             var deleted = DeleteRevisionsInternal(context, table, lowerIdPrefix, collectionName, changeVector, lastModifiedTicks, result.PreviousCount, revisionsToDelete, result);
 
+
+            NotifyAboutConflictRevisionsIfNeeded(conflicted, nonPersistentFlags, deleted, lowerIdPrefix);
+
             IncrementCountOfRevisions(context, lowerIdPrefix, -deleted);
             result.Remaining = result.PreviousCount - deleted;
             return result;
         }
 
+        private void NotifyAboutConflictRevisionsIfNeeded(bool conflicted, NonPersistentDocumentFlags nonPersistentFlags, long deleted, Slice lowerIdPrefix)
+        {
+            var lowerId = lowerIdPrefix.ToString();
+            lowerId = lowerId.Substring(0, lowerId.Length - 1); // cut the prefix last char (SpecialChars.RecordSeparator)
```

Comment:
```suggestion
            var lowerId = Encoding.UTF8.GetString(lowerIdPrefix.Content.Ptr, lowerIdPrefix.Content.Size - 1);
```
instead of allocating 2 strings?

Reply:
Doesn't work properly:
```
'docs/54\u001E\u0001ict\u0000\u0000\u0000�
m��\u0001\u0000\u0000 \u0000\u0000\u0000ocu'
```

Reply:
Maybe because it is a prefix and its end is the seperator, not the real end of the allocated memory ('lowerIdPrefix.Content.Size - 1')..